src/client/routes/index.js

```diff
@@ -48,21 +48,22 @@ export default (store) => {
       </Route>
 
       <Route onEnter={onProjectEnter}>
-        <Route path="/project/:id" component={Project}>
```

Comment:
yes, because project name is a compound with group name. For example, we have `unity` project inside `unity' folder that means repository name is `unity/unity` but not `unity`, at the same time we could have `test/unity` project. So such names bring `/` into path but router parameter `:name` matches until the first `/` - means it will match only the first level group name only. So the limitation of the react router that we couldn't match with regex which would be very helpful, this is why `splat` becomes handy for handling such way of routing.